lib/hacluster.pm

```diff
@@ -58,8 +61,10 @@ our @EXPORT = qw(
 
 # Global variables
 our $crm_mon_cmd     = 'crm_mon -R -r -n -N -1';
-our $softdog_timeout = 60;
+our $softdog_timeout = 60 * get_var('TIMEOUT_SCALE', 1);
```

Comment:
I learnt that it is better to use `bmwqemu::scale_timeout(60);` in that case.

Reply:
Yes, saw the comment in #8483 regarding `bmwqemu::scale_timeout`. Will be using that instead.